VisionFive2/NixOS/README.md

```diff
@@ -0,0 +1,184 @@
+---
+sys: NixOS
+sys_ver: null
+sys_var: null
+
+status: basic
+last_update: 2024-12-22
+---
+
+# NixOS VisionFive 2 Test Report
+
+## Test Environment
+
+### Operating System Information
+
+- System version: 25.05.20241121
+- Source code link: https://github.com/NickCao/nixos-riscv
+- Reference Installation Document: https://github.com/NickCao/nixos-riscv/README.md
+
+### Hardware Information
+
+- StarFive VisionFive 2
+- A USB-A Power Adapter
+- A USB-A to C Cable
+- A microSD Card
+- A microSD Card Reader
+- A USB to UART Debugger (e.g., CH340, CH341, FT2232, etc.)
+- Three DuPont Wires
+
+## Installation Steps
+
+### Compiling Image
+
+Run `nix build ".#hydraJobs.visionfive2"` if you happen to be using NixOS on your host.
+
+It is not recommended to install and use the `nix` package manager on other distributions. Try using e.g. the official Docker image instead:
+```shell
+docker run -ti -v .:/work ghcr.io/nixos/nix
```

Comment:
Me personally won't use `.` for `docker -v` and I won't recommend it, but anyway, it's up to you.

Reply:
> Me personally won't use `.` for `docker -v` and I won't recommend it
Then which one do you usually use? `$(pwd)`?

Reply:
> > Me personally won't use `.` for `docker -v` and I won't recommend it
> 
> Then which one do you usually use? `$(pwd)`?

Yeah, `$(pwd)` LGTM.

Reply:
Fixed by https://github.com/ruyisdk/support-matrix/pull/123/commits/d45cf3bae489d43c054c72adf10f86e8bee15554